Makefile

```diff
@@ -0,0 +1,14 @@
+all: build-test check
+
+build-test: clean
+	@docker build -t diff-shellcheck-test-image test/
+
+check:
+	@docker run -dt -v $(pwd):/home --name diff-shellcheck-test diff-shellcheck-test-image
+	@docker exec diff-shellcheck-test cd /home && bats test
```

Comment:
I'm not sure about this one, should the part after `&&` be executed inside the container or not?

Reply:
Nice catch! Yes, it should run inside the container. Fixed